benchmarks/latency.dart

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) 2021, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:cronet/cronet.dart';
+
+class CronetBenchmark {
```

Comment:
LatencyBenchmark with subclasses CronetLatencyBenchmark and DartIOLatencyBenchmark.

Reply:
The only difference between two will be swapping the `import` statement. But, how can I achieve this without 2 near duplicate file? How can I make this subclass structure?

(Maybe it's a really basic question to ask :sweat_smile: )